src/ILLink.RoslynAnalyzer/RequiresAnalyzerBase.cs

```diff
@@ -183,7 +183,7 @@ public override void Initialize (AnalysisContext context)
 							if (instanceCtor.Arity > 0)
 								continue;
 
-							if (instanceCtor.TargetHasRequiresAttribute (RequiresAttributeName, out var requiresAttribute) &&
+							if (instanceCtor.DoesMemberRequires (RequiresAttributeName, out var requiresAttribute) &&
```

Comment:
Nit - not for this PR: The grammar of the new name is really weird - "Does member requires" feels wrong, I think it should read "Does member require <something>". I know it's how we have it everywhere, so not for this change, but maybe something to consider one day :-)